spec/models/article_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe Article, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :title }
+    it { is_expected.to have_db_column :lede }
+    it { is_expected.to have_db_column :body }
+    it { is_expected.to have_db_column :url }
+    it { is_expected.to have_db_column :date }
```

Comment:
What is the `date` attribute used for in this context?

Reply:
The `date` has been removed.